resources/views/PageDesign/Partials/Head.twig

```diff
@@ -33,7 +33,11 @@
 {% set itemName = config('Ceres.item.name') %}
 
 {% if LayoutContainer.show("Ceres::Template.StyleOverwrite") | trim is empty %}
-    <link rel="stylesheet" href="{{ plugin_path('Ceres') }}/css/ceres.css">
+    {% if config('plugin.setId', null) != null %}
```

Comment:
@felixdausch maybe better to use `{% if config('plugin.setId', null) is not empty %}` here?